### PR TITLE
Refactor/creation-flow

### PR DESCRIPTION
**Pull request by sajald77**
resolves GEY-5638, GEY-5308, GEY-5519, GEY-5419, GEY-5309, GEY-5558

### CODE REVIEW

**Review by travishoglund — CHANGES_REQUESTED** [review]
Fantastic work on this - I added some comments, but it looks really good

File: src/components/ui/TextArea.tsx [hunk 1]
```diff
@@ -12,8 +12,8 @@ const useStyles = createUseStyles(({ colors }: AppTheme) => ({
     borderRadius: '8px',
     borderWidth: '2px',
     '&:focus': {
-      borderColor: `${colors.neutral[500]} !important`,
-      boxShadow: `0 0 0 1px ${colors.neutral[500]}`,
+      borderColor: `${colors.primary[500]} !important`,
```
[the remark this answers is not in the document]
Comment (sajald77) [reply src/components/ui/TextArea.tsx:15]: Yes, I got that, and yes it is intended. I checked all the places it was used, and was surprised why it is not standard like the designs.

File: src/components/ui/TextInputBox.tsx [hunk 2]
```diff
@@ -41,9 +41,9 @@ export const TextInputBox = forwardRef<HTMLInputElement, TextInputBoxProps>(
             _hover={{
               borderColor: 'neutral.400',
             }}
-            _focus={{ borderColor: `neutral.500`, boxShadow: 'none' }}
+            _focus={{ borderColor: `primary.500`, boxShadow: 'none' }}
```
[the remark this answers is not in the document]
Comment (sajald77) [reply src/components/ui/TextInputBox.tsx:44]: same as above

File: src/pages/projectCreate/ProjectCreateStart.tsx [hunk 3]
```diff
@@ -140,9 +144,7 @@ export const ProjectCreateStart = () => {
             </Text> */}
           </VStack>
         ) : (
-          <Box w="100%">
-            <FormContinueButton width="100%" onClick={handleNext} />
-          </Box>
+          <Box w="100%"></Box>
```
[the remark this answers is not in the document]
Comment (sajald77) [reply src/pages/projectCreate/ProjectCreateStart.tsx:147]: I think that needs removing, nice catch.

**Review by travishoglund — APPROVED** [review]

**Pull request merged**